docs/maintainer/infrastructure.md

```diff
@@ -2,28 +2,155 @@
+
+#### conda-forge pinning
+
+Hosts the global pinnings for conda-forge, and the ongoing migrations.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-pinning) via [`conda-forge/conda-forge-pinning-feedstock`](https://github.com/conda-forge/conda-forge-pinning-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+Package-wide dependency pins are defined in [conda_build_config.yaml](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml) in the [conda-forge/conda-forge-pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock).
+
+For more information on conda-forge wide package pins, please refer to [Globally pinned packages](pinning_deps.md#globally-pinned-packages).
+
+Please open a [PR](../glossary.md#term-PR) and/or an issue there, if you think a pin needs to be advanced. For more information on updating globally pinned packages, please refer to [Updating package pins](pinning_deps.md#update-pins).
+
+#### conda-forge-repodata-patches
+
+This repository creates the `repodata.json` patches used by the Anaconda.org to amend the metadata coming from the published packages.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-repodata-patches) via [`conda-forge/conda-forge-repodata-patches-feedstock`](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+#### conda-forge-ci-setup
+
+This special feedstock provides a package that defines the logic to install and configure a common CI setup across providers.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-ci-setup) via [`conda-forge/conda-forge-ci-setup-feedstock`](https://github.com/conda-forge/conda-forge-ci-setup-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+#### regro/cf-graph-countyfair
 
-<a id="smithy"></a>
+This is the graph data used by [`autotick-bot`](#autotick-bot).
 
-### Smithy
+- ⚙️ Deployed in [Github Actions via `regro/cf-graph-countyfair`](https://github.com/regro/cf-graph-countyfair)
+- ⛓ Needs [`regro/cf-scripts`](#regrocf-scripts), [`conda-forge/conda-forge-pinning-feedstock`](#conda-forge-pinning)
+- 🤖 Uses [`@regro-cf-autotick-bot`](#bot-accounts)
+- 🔒 Has access to Github API
 
-Smithy contains maintenance code for conda-forge, which is used by the `conda-smithy` command line tool and the [Admin web services](#dev-admservice). Smithy lives in the repository [conda-forge/conda-smithy](https://github.com/conda-forge/conda-smithy).
+The logic to build the graph is provided by [`cf-scripts`](#regrocf-scripts).
+
+#### regro/conda-suggest-conda-forge
+
+- [`regro/conda-suggest-conda-forge`](https://github.com/regro/conda-suggest-conda-forge) provides [`conda-suggest`](https://github.com/conda-incubator/conda-suggest) files that map executables to package names.
+
```

Comment:
As far as I know, this is a private-ish tool built by @scopatz and not so much a semi-official conda-forge thing. It might make sense to not list it here.